cargo-audit/tests/acceptance.rs

```diff
@@ -26,9 +26,9 @@ static ADVISORY_DB_DIR: Lazy<TempDir> = Lazy::new(|| TempDir::new().unwrap());
 /// be multithreaded invocations as `cargo test` executes tests in
 /// parallel by default.
 pub static RUNNER: Lazy<CmdRunner> = Lazy::new(|| {
-    let mut runner = CmdRunner::default();
+    let mut runner = CmdRunner::new(env!("CARGO_BIN_EXE_CARGO_AUDIT"));
```

Comment:
What is this environment variable?

Reply:
It seems this should be using `std::env::var_os` instead of `env!`. 

I doubt this is going to be any faster, but I'd be happy to be proven wrong.

Reply:
Panic - it's supposed to be always set: https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

Reply:
Nevermind, I got it all wrong. It should be `env!(CARGO_BIN_EXE_cargo-audit)`